### PR TITLE
fun with functions complete

### PR DESCRIPTION
**Pull request by schmittkw**

### CODE REVIEW

**Comment by schmittkw** [comment]
scores n grades completed

**Comment by schmittkw** [comment]
coin toss complete

**Comment by schmittkw** [comment]
fun functions update

**Comment by schmittkw** [comment]
stars complete

**Comment by schmittkw** [comment]
dictionary basics completed

**Comment by schmittkw** [comment]
bike complete

File: kevin_schmitt/bike.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            print 'Now i have ridden {} miles since i have gone backwards!'.format(self.miles)
+        return self
+    
+bike1 = Bike('Schwinn', '300','32', '500')
+bike2 = Bike('Huffy',  '201','28','600')
+print bike1.max_speed
+print bike2.price
+bike2.displayinfo()
+bike1.ride()
+bike1.reverse()
```
Comment (codyw9524) [inline kevin_schmitt/bike.py:29]: 👍

**Pull request merged**